### PR TITLE
Attemp to fix pydantic dependency issue

### PR DESCRIPTION
**Pull request by mars-lan**
### 🤔 Why?

Installing from the latest PyPI package does not include `pydantic`.

### 🤓 What?

Dropping `pydantic` from dbt extra to see if it helps.

### 🧪 Tested?

N/A

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -47,7 +47,7 @@ all = [
   "tableauserverclient",
 ]
 bigquery = ["google-cloud-bigquery", "google-cloud-logging"]
-dbt = ["pydantic"]
+dbt = []
```
Comment (alyiwang) [inline pyproject.toml:50]: can remove this line?

**Review by alyiwang — COMMENTED** [review]
A similar fix, with updated dbt README 
https://github.com/MetaphorData/connectors/pull/153

File: pyproject.toml (hunk 1 above)
Comment (mars-lan) [reply pyproject.toml:50]: This will remove the `dbt` extra

**Comment by mars-lan** [comment]
> A similar fix, with updated dbt README #153

Prefer to keep the empty extra for consistency

**Comment by mars-lan** [comment]
Closing in favor of https://github.com/MetaphorData/connectors/pull/153

**Pull request closed without merging**